opm-common-prereqs.cmake

```diff
@@ -25,3 +25,7 @@ else()
         "Boost 1.44.0
           COMPONENTS system unit_test_framework REQUIRED")
 endif()
+# We need a defined target for libecl when linking.
+# The definition of the target is done implicitly below when
+# libecl is searched for.
+find_package_deps(opm-common)
```

Comment:
Do you really mean `opm-common` here (in `opm-common-prereqs.cmake`)?  Is `opm-common` a prerequisite for itself?

Reply:
it's the prefix for the variables used inside the macros. opm-common_CONFIG_VAR, opm-common_DEPS.

Reply:
> it's the prefix for the variables used inside the macros

Ah, right.  Thanks for clarifying and sorry for the confusion.